src/vue/backend/src/views/GuidedSetupGeneral.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <GuidedSetupContentCard
+        :cardTitle="cardTitle"
+        :steps="guidedSetupSteps"
+        :relatedLinks="links"
+        @back-step="goBackStep"
+        @skip-step="goSkipStep"
+        @continue="goContinue">
+        <template v-slot:controls>
+            <GeneralSettings/>
+        </template>
+    </GuidedSetupContentCard>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex'
+import GeneralSettings from '@/templates/GeneralSettings.vue'
+import GuidedSetupContentCard from '@/templates/GuidedSetupContentCard.vue'
+export default {
+    name: 'GuidedSetupGeneral',
+    components: {
+        GeneralSettings,
+        GuidedSetupContentCard
+    },
+    data () {
+        return {
+            cardTitle: 'General Settings',
+            links: [
+                {
+                    text: 'Dynamic Wrappers',
+                    href: '#dynamic-wrappers'
+                },
+                {
+                    text: 'IDX Broker Middleware',
+                    href: 'https://middleware.idxbroker.com/mgmt/'
+                },
+                {
+                    text: 'Sign up for IDX Broker',
+                    href: '#signUp'
+                }
+            ]
+        }
+    },
+    computed: {
+        ...mapState({
+            guidedSetupSteps: state => state.general.guidedSetupSteps
+        })
+    },
+    methods: {
+        ...mapActions({
+            generalSettingsStateChange: 'general/generalSettingsStateChange',
+            verifyAPIkey: 'general/verifyAPIkey'
+        }),
+        goBackStep: function () {
+            // to-do: go back in history
+            this.$router.go(-1)
+        },
+        goSkipStep: function () {
+            this.$router.push({ path: '/guided-setup/connect/omnibar' })
+        },
+        async goContinue () {
+            await this.verifyAPIkey()
```

Comment:
Why are you verifying the API key? There is not an api key for this page in the designs.

Reply:
in the A/C it said:
@continue

- Ping API endpoint: TBD

Reply:
Correct, ping the corresponding api endpoint. Namely, the one that will send the form change to the db. Currently, you are using an action that is specifically for verifying the `APIKey`, which is not included in this ticket.